### ISSUE TITLE
Saving model issue

### ISSUE THREAD

**Issue by lr13953301387** [hunglc007/tensorflow-yolov4-tflite#332]
I suppose model.save_weights("./checkpoints/yolov4") is going generate a .pb file in my directory, but I can only find a *.data-00000-of-00001 file, a *.index file and a file called checkpoints. Anyone know what might go wrong?

**Comment by RodolfoFerro** [comment]
You may want to add `model.save('./checkpoints/yolov4')` after that line and it will create a new folder as follows:

```bash
└── yolov4
    ├── assets
    ├── saved_model.pb
    └── variables
        ├── variables.data-00000-of-00001
        └── variables.index
```

**Comment by lr13953301387** [comment]
> You may want to add `model.save('./checkpoints/yolov4')` after that line and it will create a new folder as follows:
> 
> ```shell
> └── yolov4
>     ├── assets
>     ├── saved_model.pb
>     └── variables
>         ├── variables.data-00000-of-00001
>         └── variables.index
> ```

Thank you for your reply, this indeed solves the problem. However, I encounter another problem. I have trained my model for around 10 epochs and the loss has dropped into a relatively low value. Then I load my model and train it again, but the weights of my previous trained model seem like have not been loaded because the loss value is much higher. Do you have any idea what might cause it?

Thanks
